### PR TITLE
add whitelists per feature

### PR DESCRIPTION
**Pull request by nmushegian**
to be replaced with group auth system...

### CODE REVIEW

**Comment by nmushegian** [comment]
@dbrock @jorisbontje pls review

**Comment by nmushegian** [comment]
add @rainbeam

File: contracts/ss.sol [hunk 1]
```diff
@@ -140,9 +151,41 @@ contract SimpleStablecoin is ERC20Base(0) {
...
 
+    modifier issuers_only() {
+        if( _issuer_whitelist.isWhitelisted(msg.sender) ) {
+            _
+        } else {
+            throw;
+        }
+    }
+
+    modifier transfer_whitelist(address who) {
```
Comment (jorisbontje) [inline contracts/ss.sol:162]: `who` is unused

File: contracts/ss.sol [hunk 2]
```diff
@@ -140,9 +151,41 @@ contract SimpleStablecoin is ERC20Base(0) {
...
+        if( _transfer_whitelist.isWhitelisted(msg.sender) ) {
```
Comment (jorisbontje) [inline contracts/ss.sol:163]: `msg.sender` should probably be `who`

File: contracts/ss.sol [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
-    function feedbase() constant returns (address) { return _feedbase; }
-    function type_count() constant returns (uint) { return _types.length; }
+    // uses two whitelists instead of a GroupAuthority... for now
+    Whitelist _issuer_whitelist;
+    Whitelist _transfer_whitelist;
 
+    uint _exchange_price; // Number of wei for each 10**18 of your token
+    uint _exchange_spread; // +- on price
+    uint64 _exchange_expires; // don't trade when price info is stale
+    uint _max_supply; // don't create more than this many stablecoins
```
Comment (jorisbontje) [inline contracts/ss.sol:22]: unused properties, merge problem?

**Comment by rainbreak** [comment]
LGTM pending comments above. What's your thinking on group auth? Is that something to be added to ds-whitelist?

**Pull request merged**